docs/tutorials/basic_networking.md

```diff
@@ -13,14 +13,13 @@ Each setup is supported with an example.
 
 ## Differences between rootful and rootless container networking
 
-One of the guiding factors on networking for containers with Podman is going to be
-whether or not the container is run by a root user or not.  This is because unprivileged
-users cannot create networking interfaces on the host.  Therefore, with rootful
-containers, the default networking mode is to use netavark.
-For rootless, the default network
-mode is slirp4netns. Because of the limited privileges, slirp4netns lacks some of
-the features of networking; for example, slirp4netns cannot give containers a
-routable IP address.
+One of the guiding factors on networking for containers with Podman is going to be 
+whether or not the container is run by a root user or not. This is because unprivileged
+users cannot create networking interfaces on the host. Therefore, for rootless containers, 
+the default network mode is slirp4netns. Because of the limited privileges, slirp4netns 
+lacks some of the features of networking; for example, slirp4netns cannot give containers 
+a routable IP address. With rootful containers, on the other side, the default networking
+mode is to use netavark.
```

Comment:
I'd simplify the last sentence further.

The default networking mode for rootful containers is netavark.

or if you wanted to make it a bit fuller:

The default networking mode for rootful containers is netavark, which allows a container to have a routable IP address.

Reply:
I incorporated this in my branch (vrms/patch-1). I don't think I need to do more with the PR now, do I?